spec/views/users/users_index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Tomy',
+                         photo: 'https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg',
+                         bio: 'Tomy bio',
+                         posts_counter: 0)
+    @user2 = User.create(name: 'Lima',
+                         photo: 'https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg',
+                         bio: 'Lima bio',
+                         posts_counter: 0)
+
+    Post.create(title: 'Post 1', text: 'Text 1', author: @user1, comments_counter: 0, likes_counter: 0)
+    Post.create(title: 'Post 2', text: 'Text 2', author: @user2, comments_counter: 0, likes_counter: 0)
+
+    visit users_path
+  end
+
+  it 'Should display the user page heading' do
+    expect(page).to have_content('Microverse Community - All Users')
+  end
+
+  describe 'Displaying user information' do
+    it 'Should display names of all the users' do
+      expect(page).to have_content('Tomy')
+      expect(page).to have_content('Lima')
+    end
+    it 'Should display user photos' do
+      expect(page).to have_css(
+        "img[src*='https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg']",
+        visible: :visible
+      )
+      expect(page).to have_css(
+        "img[src*='https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg']", visible: :visible
+      )
+    end
+    it 'Should display users post counter' do
+      expect(page).to have_selector('.num-posts', count: 4) # Assuming there are two users with post counters
+    end
```

Comment:
- [ Required ] I think it would be nice if you fix this test, I think since your spec file has only two posts the number of posts that you should test for needs to be `2` and not `4`. Please see the screenshot below and rectify this.

## Error when running tests
![Screenshot from 2023-09-09 01-05-08](https://github.com/MoyasiGinko/Rails-Blog/assets/16392046/ab01aa7e-6505-4d56-baaa-e2d91fe7ea0a)

Reply:
hey @Felix45 , can you please show me how to solve this issue?